apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,37 @@
 # API Changelog
 
+## BFD-1518
```

Comment:
I'd expand on this title with high level what was changed. Looks like it's written below